packages/fxa-settings/src/lib/auth-errors/en-US.ftl

```diff
@@ -2,7 +2,7 @@
 
 auth-error-102 = Unknown account
 auth-error-103 = Incorrect password
-auth-error-105 = Invalid verification code
```

Comment:
These do seem to correspond to errors of the same code (in `packages/fxa-settings/src/lib/auth-errors/auth-errors.ts`) but I don't know where these IDs are used. Since I didn't turn them up in a simple vscode search, I thought it might be possible that they're being composed from the error code, in which case this incrementation method might not work.

Reply:
Figured this out. The ids assigned to the strings are composed in the code (like, `auth-error-${e.errno}`). Since we now (sometimes but not always) will have to append a version number also, I've broken that out into a small utility function, and also added a `version` key value pair to the `ERRORS` const in `auth-errors/auth-errors.ts`.